### PR TITLE
san diego contributor summit blog

### PR DESCRIPTION
**Pull request by parispittman**
blog post for the contributor summit

### CODE REVIEW

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Contributor Summit San Deigo Registration Open!"
+date: 2019-09-23
+slug: san-deigo-contributor-summit
```
Comment (jonasrosland) [inline content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: This should be `san-diego-contributor-summit`

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Contributor Summit San Deigo Registration Open!"
```
Comment (jonasrosland) [inline content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: `San Diego`

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+date: 2019-09-23
+slug: san-deigo-contributor-summit
+---
+
+**Authors: Paris Pittman (Google), Jeffrey Sica (Red Hat), Jonas Rosland (VMWare)**
```
Comment (jonasrosland) [inline content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: Lower case w, `VMware` :)

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Check out past blogs on [persona building around our events] and the [Barcelona summit story].
+
+[Contributor Summit San Diego 2019 Event Page]: https://events.linuxfoundation.org/events/kubernetes-contributor-summit-north-america-2019/
+[QuartYard]: https://quartyardsd.com/
+[Marriott Marquis San Diego Marina]: https://www.marriott.com/hotels/travel/sandt-marriott-marquis-san-diego-marina/?scid=bb1a189a-fec3-4d19-a255-54ba596febe2
+[individuals and organizations]: https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=All&var-companies=All
+[rolebooks, guidelines, best practices]: https://github.com/kubernetes/community/tree/master/events/events-team
+[meetings]: https://docs.google.com/document/d/1oLXv5_rM4f645jlXym_Vd7AUq7x6DV-O87E6tcW1sjU/edit?usp=sharing
+[project board]: https://github.com/orgs/kubernetes/projects/21
+[SIG Intro and Deep Dive sessions]: https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2019/schedule/
```
Comment (jonasrosland) [inline content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: If you want to link to the Maintainer Track Sessions, you can use this: https://kccncna19.sched.com/overview/type/Maintainer+Track+Sessions?iframe=yes

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+Are you at KubeCon + CloudNativeCon but can’t make it to the summit? Check out
+the [SIG Intro and Deep Dive sessions] under the 'maintainer track' label during
+KubeCon + CloudNativeCon to participate in Q&A and hear what’s up with each
+Special interest Group (SIG). We’ll also record all of Contributor Summit’s
+presentation sessions, take notes in discussions, and share it back with you,
+after the event is complete.
+
+We hope to see you all at Kubernetes Contributor Summit San Diego, make sure you
+head over and [register right now]! This event will sell out - here’s your warning.
+:)
```
Comment (sftim) [inline content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: Would a :smile: (in Unicode) work better here?

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 1 above)
Comment (parispittman) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: corrected

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 2 above)
Comment (parispittman) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: corrected

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 3 above)
Comment (parispittman) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: corrected

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+all forms of diversity.
+
+Kubernetes has a large distributed remote contributing team, from [individuals and
+organizations] all over the world. The Contributor Summits give the community three
+chances a year to get together, work on community topics, and have hallway track
+time. The upcoming San Diego summit is expected to bring over 450 attendees, and
+will contain multiple tracks with something for everyone. The focus will be around
+contributor growth and sustainability. We're going to stop here with capacity for
+future summits; we want this event to offer value to individuals and the project.
+Getting work done and learning is hard in groups larger than this but it's a priority
```
[the remark this answers is not in the document]
Comment (parispittman) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: reworded - lmk

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 4 above)
Comment (parispittman) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: added

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 5 above)
Comment (parispittman) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: added :grin:

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit [hunk 7]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Contributor Summit San Diego Registration Open!"
+date: 2019-09-23
```
Comment (kbhawkey) [inline content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: hello @parispittman .
Is the front matter for this page missing the layout field, such as:

```
---
layout: blog
title: "Kubernetes 1.16: Custom Resources, Overhauled Metrics, and Volume Extensions"
date: 2019-09-18
slug: kubernetes-1-16-release-announcement
---
```
Would it be helpful to create a template to populate the required front matter for blog pages?

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 6 above)
[the remark this answers is not in the document]
Comment (jonasrosland) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: Much better, thank you!

File: content/en/blog/_posts/2019-09-23-san-diego-contributor-summit (hunk 7 above)
Comment (jonasrosland) [reply content/en/blog/_posts/2019-09-23-san-diego-contributor-summit]: I think this is the only thing missing from the blog right now @parispittman, then we can roll it out.

**Comment by mrbobbytables** [comment]
/hold
some quick fixes in while getting an approve

**Comment by mrbobbytables** [comment]
/lgtm

release the hold when ready 👍

**Comment by kbhawkey** [comment]
Looks good. Is the branch up to date? I don't see the list of currently published blogs in the right hand nav? I may be looking at an old preview though.

**Pull request merged**